src/sentry/templates/sentry/emails/incidents/trigger.txt

```diff
@@ -0,0 +1,22 @@
+{% spaceless %}
+{% autoescape off %}
+# Alert Rule Trigger {{ status }} on Incident {{ incident_name }}.
+
+{% if enhanced_privacy %}
+Details about this incident alert are not shown in this email since enhanced privacy
+controls are enabled. For more details about this incident alert, view on Sentry:
+{{ incident_link }}.
+{% else %}
+Incident: {{ link }}
+Rule: {{ rule_link }}
+
+Aggregate: {{ aggregate }}
```

Comment:
Will aggregate make sense to the user? Should this be metric?

Reply:
That probably makes sense. These are just rough emails at the moment, someone from design/product will decide what these should look like.